etc/docker/Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM debian:bullseye
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NOWARNINGS="yes"
+
+ENV RUSTUP_HOME=/opt/rust/rustup \
+    CARGO_HOME=/opt/rust/cargo \
+    PATH=/opt/rust/cargo/bin:$PATH \
+    RUST_VERSION=1.65.0 \
+    ESPRS_VERSION=v1.65.0.0 \
+    ESP_ROOT=/opt/esp \
+    IDF_TOOLS_PATH=/opt/esp/toolchain
+
+RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list.d/backports.list
```

Comment:
You can also do this with `apt-add-repository` from `software-properties-common`:
- https://github.com/viamrobotics/viam-cpp-sdk/blob/961544fa1490c3514f26eae7c255d46a7703fe9f/etc/docker/Dockerfile.debian.bullseye#L27
- https://github.com/viamrobotics/viam-cpp-sdk/blob/961544fa1490c3514f26eae7c255d46a7703fe9f/etc/docker/Dockerfile.debian.bullseye#L31